cluster/okd/manifests/validating-webhook.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: admissionregistration.k8s.io/v1beta1
+apiVersion: admissionregistration.k8s.io/v1
```

Comment:
Some of the fields have changed between versions `v1betav1` and `v1`.
The `v1` version requires these fields:
- SideEffects
- AdmissionReviewVersions

You can copy the values from ssp-operator